integration-tests/bats/sql-local-remote.bats

```diff
@@ -283,4 +283,21 @@ get_staged_tables() {
     [[ "$output" =~ "starting local mode" ]] || false
     [[ "$output" =~ "main" ]] || false
     [[ "$output" =~ "b2" ]] || false
-}
\ No newline at end of file
+}
+
+@test "sql-local-remote: verify simple dolt reset behavior" {
```

Comment:
This test doesn't fit the pattern of other tests in this file. 1) You need to stop the server, and verify that the steps you run in both contexts are identical. 2) don't bother passing in `--verbose-engine-setup` if you aren't going to check the output. You should only  pass this to calls of `dolt reset` - since that is the command under observation.